### PR TITLE
Cinco de yakko

### PR DESCRIPTION
**Pull request by yaleman**
Refers to #1987

Checklist

- [x] This pr contains no AI generated code
- [x] cargo fmt has been run
- [x] cargo clippy has been run
- [x] cargo test has been run and passes

### CODE REVIEW

File: server/lib/src/entry.rs [hunk 1]
```diff
@@ -159,16 +159,22 @@ pub struct EntryReduced {
     uuid: Uuid,
 }
 
+// One day this is going to be Map<Attribute, ValueSet> - @yaleman
+// pub type Eattrs = Map<Attribute, ValueSet>;
```
Comment (Firstyear) [inline server/lib/src/entry.rs:163]: How will that work with custom attributes that users define? Will we need a way for them to make and assign there or wrap these somehow?

**Review by Firstyear — APPROVED** [review]

File: server/lib/src/entry.rs (hunk 1 above)
Comment (yaleman) [reply server/lib/src/entry.rs:163]: Yeah, I figure we'll have a `Attribute::CustomAttribute(String)` or something like that?

Probably good to auto-prepend "custom_" on the name when they're created, and maybe hide it when displaying, or something?

**Comment by yaleman** [comment]
I'll hold this one and rebase it once you're done with the attrs thing you mentioned.

**Comment by Firstyear** [comment]
That's merged now, so you should be able to rebase :)

**Pull request merged**